.travis.yml

```diff
@@ -0,0 +1,17 @@
+sudo: required
+dist: trusty
+language: cpp
```

Comment:
I haven't looked at travis lately. I'm guessing there's no plain "c", if you've specified cpp here? memcached is pure C.

Reply:
oh sorry, it's my mistake. actually, there is a plan whose language is c, but the builder can compile pure c project successfully, because it will choose gcc instead of g++ to compile the project in c.